### PR TITLE
Airbnb Search bar

### PR DESCRIPTION
**Pull request by Dimoshua**
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Comment by Dimoshua** [comment]
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-style: normal;
+  font-weight: 800;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("truetype");
+  font-style: normal;
+  font-weight: 600;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:18]: image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+html {
+  font-family: Avenir-Book, Avenir-Heavy, Avenir, Arial, Helvetica, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:21]:
```suggestion
  font-family: Avenir, Arial, Helvetica, sans-serif;
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Fix icon size 
image
Remove black outline

**Pull request closed without merging**